.github/workflows/maintenance-cherry-pick-workflow.yml

```diff
@@ -0,0 +1,40 @@
+name: cherry-pick on maintenance branch
+
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  cherry_pick:
+    runs-on: ubuntu-latest
+    outputs:
+      output: ${{ steps.cherry_pick.outputs.output }}
+    if: |
+      github.event.pull_request.merged == true &&
+      contains(github.event.pull_request.labels.*.name, 'target: maintenance')
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: "Cherry-pick changes into maintenance"
+      id: cherry_pick
+      run: |
+        git checkout maintenance
+        echo "::set-output name=output::$(git -c user.name="cherry.picker" \
+          -c user.email="cherry.picker@sbb.ch" cherry-pick ${{ github.sha }} | xargs)"
```

Comment:
I do wonder whether it is wise to use a mail address that does not exist.
Maybe just use `user.name="github-actions" user.email="github-actions@github.com"`

See https://github.com/actions/checkout#push-a-commit-using-the-built-in-token